charts/astronomer/templates/commander/commander-deployment.yaml

```diff
@@ -143,7 +143,7 @@ spec:
             - name: COMMANDER_BASE_DOMAIN
               value: {{ .Values.global.baseDomain | default "" | quote }}
             - name: COMMANDER_DATAPLANE_MODE
-              value: {{ .Values.global.dataplane.enabled | default false | quote }}
+              value: {{ if eq .Values.global.plane.mode "data" }}"true"{{ else }}"false"{{ end }}
```

Comment:
we no longer support boolean - we can directly send the dataplane mode types like control, data, or unified